lib/modal/theme.module.scss

```diff
@@ -101,15 +124,16 @@
 }
 
 :local(.close) {
-    width: fit-content;
-    cursor: pointer;
-    position: absolute;
-    right: 2%;
     display: flex;
-    align-items: center;
+    width: fit-content;
     height: fit-content;
+    position: absolute;
     font-size: 20px;
     color: #1f1e1eab;
+    right: 0.5em;
```

Comment:
can be changed to `1em`

Reply:
Updated in https://github.com/Codebrahma/react-lite-ui/pull/295/commits/efc1ceeb988e53662c2e67789757eb26eef01261. Also added text clipping for long header texts.